config/routes.rb

```diff
@@ -22,10 +22,13 @@
   namespace :api do
     namespace :v1 do
       jsonapi_resources :jobs do
+        jsonapi_relationships
```

Comment:
Should probably look into the full impact of adding this (I know this is my code!)